cli/tools/registry/pm/deps.rs

```diff
@@ -613,6 +614,9 @@ impl DepManager {
       )
       .await?;
 
+    self
+      .dependencies_resolved
+      .store(true, std::sync::atomic::Ordering::Relaxed);
```

Comment:
We should change this to use `SyncFlag` sometime in the future as the api is slightly nicer.